.github/workflows/time-machine.yml

```diff
@@ -182,8 +182,7 @@ jobs:
         if:   matrix.date-offset != '0 years'
```

Comment:
@ni4  This condition is redundant after your changes

Reply:
@maxirmx Thanks for noticing, fixed.